pre_commit/clientlib/validate_manifest.py

```diff
@@ -17,6 +17,7 @@ class InvalidManifestError(ValueError):
         'type': 'object',
         'properties': {
             'id': {'type': 'string'},
+            'always_run': {'type': 'boolean'},
```

Comment:
we can `'default': False` here and then the check for this gets simpler in pre_commit/run.py